File: Domain/Entities/ET/DocumentApproved.cs
using System;
using System.Collections.Generic;

namespace Domain.Entities.ET
{
    public class DocumentApproved : EntityBase
    {
        public string EmployeeCode { get; set; }
        public Guid? EvaluationStatus { get; set; }
        public Guid? DocumentNo { get; set; }
        public string ApproveBy { get; set; }
        public ICollection<DocumentApprovedDetail> DocumentApprovedDetails { get; set; }
        public ICollection<DocumentApprovedSkill> DocumentApprovedSkills { get; set; }
    }
}

File: Web/ClientApp/src/app/models/et/documentApproved.ts
import { Guid } from "guid-typescript";
import { EntityBase } from "../entityBase";
import { DocumentApprovedDetail } from "./documentapprovedDetail";
import { DocumentApprovedSkill } from "./documentapprovedSkill";

export class DocumentApproved extends EntityBase {
    employeeCode: string;
    evaluationStatus: Guid;
    documentNo: Guid;
    approveBy: string;
    documentapprovedDetails : DocumentApprovedDetail[]
    documentapprovedSkills : DocumentApprovedSkill[]

    constructor(){
      super();
      this.documentapprovedDetails = []
      this.documentapprovedSkills = []
    }
}
